src/main/java/org/infernalstudios/shieldexp/init/ShieldDataLoader.java

```diff
@@ -83,12 +85,47 @@ public static void parse(ResourceLocation name, JsonObject data) {
             SHIELD_STATS.remove(key);
             SHIELD_STATS.put(key, stats);
 
-            if (!key.equals(ShieldExpansion.MOD_ID + ":default"))
+            if (!key.equals(DEFAULT_SHIELD_NAME))
                 Config.extendList(key);
         }
     }
 
     public static void clearAll() {
         SHIELD_STATS.clear();
     }
+
+    private static class ShieldStatsMap extends HashMap<String, Map<String, Double>> {
+        // This should NEVER need to be used, but just in case the data is missing, this is the hard-coded default data
+        private static final Map<String, Double> EMERGENCY_DEFAULT = new HashMap<>() {
+            {
+                // these should be the same as the ones in src/resources/data/shieldexp/shields/default.json
+                this.put("cooldownTicks", 30.0);
+                this.put("speedFactor", 0.65);
+                this.put("parryDamage", 0.10);
+                this.put("parryTicks", 5.0);
+                this.put("stamina", 2.0);
+                this.put("blastResistance", 0.0);
+                this.put("flatDamage", 1.0);
+            }
+
+            // in case this map ever gets used, log a fatal error
+            @Override
+            public Double get(Object key) {
+                ShieldExpansion.LOGGER.fatal("The client is missing shield data from the server! This includes the default shield data! Please report this bug to Infernal Studios.");
```

Comment:
This is not a fatal error, or an error in general, we can recover from this, it should be a warning:
```suggestion
                ShieldExpansion.LOGGER.warn("The client is missing shield data from the server! This includes the default shield data! Please report this bug to Infernal Studios.");
```